### ISSUE TITLE
docker-compose.yaml comma-separate validators

### ISSUE THREAD

**Issue by ladidan** [TobiWo/eth-duties#38]
Hi folks,
cool tool!

This is just a quick suggestion for the compose.yaml. I'd argue it is much leaner to comma-separate validator indices here, e.g.

```
        container_name: eth-duties
        restart: on-failure
        command:
        - --validators
        - "1234,12345,123456"
        - --beacon-node
        - http://localhost:5052
```

**Comment by ladidan** [comment]
Ah ok, good to know, I initially tested in on v0.2.1 and comma-separation worked fine

**Comment by TobiWo** [comment]
It is a bit strange how the Python argument parser works here. I needed to change the behavior of the flag so that you can pass multiple `--validators` flags but this led to the fact that comma separation is not longer supported. And in the compose on the other hand validator space separation doesn't work unfortunately. My advice would be to use a validators-file and mount that into the container if you have many validators and working with a compose. I think up to 5 indices/pubkeys where each is placed on a new line is ok but if you much more than that I would use a file instead.

**Comment by TobiWo** [comment]
@ladidan Just as an update on this topic. I will implement a custom cli flag parsing function to allow also comma separation for validator indices/pubkeys again. Actually not allowing comma separation has a lot of side effects which is super bad. I'm off for a couple of days but will fix this asap.

**Comment by TobiWo** [comment]
@ladidan This issue will be closed soon through PR #40 which will lead rekease version v0.3.1. If your issues remain or you find new stuff just re-open the issue or open a new one. Thanks.
